crates/store/re_types/definitions/rerun/archetypes/scalar.fbs

```diff
@@ -1,22 +1,21 @@
 namespace rerun.archetypes;
 
-// ---
-
 /// A double-precision scalar, e.g. for use for time-series plots.
 ///
 /// The current timeline value will be used for the time/X-axis, hence scalars
-/// cannot be static.
+/// should not be static.
 ///
 /// When used to produce a plot, this archetype is used to provide the data that
-/// is referenced by [archetypes.SeriesLine] or [archetypes.SeriesPoint]. You can do
+/// is referenced by [archetypes.SeriesLines] or [archetypes.SeriesPoints]. You can do
 /// this by logging both archetypes to the same path, or alternatively configuring
 /// the plot-specific archetypes through the blueprint.
 ///
-/// \example archetypes/scalar_simple !api title="Simple line plot" image="https://static.rerun.io/scalar_simple/8bcc92f56268739f8cd24d60d1fe72a655f62a46/1200w.png"
-/// \example archetypes/scalar_multiple_plots !api title="Multiple time series plots" image="https://static.rerun.io/scalar_multiple/15845c2a348f875248fbd694e03eabd922741c4c/1200w.png"
-/// \example archetypes/scalar_row_updates title="Update a scalar over time" image="https://static.rerun.io/transform3d_column_updates/2b7ccfd29349b2b107fcf7eb8a1291a92cf1cafc/1200w.png"
-/// \example archetypes/scalar_column_updates title="Update a scalar over time, in a single operation" image="https://static.rerun.io/transform3d_column_updates/2b7ccfd29349b2b107fcf7eb8a1291a92cf1cafc/1200w.png"
+/// \example archetypes/scalars_simple !api title="Simple line plot" image="https://static.rerun.io/scalar_simple/8bcc92f56268739f8cd24d60d1fe72a655f62a46/1200w.png"
+/// \example archetypes/scalars_multiple_plots !api title="Multiple time series plots" image="https://static.rerun.io/scalar_multiple/15845c2a348f875248fbd694e03eabd922741c4c/1200w.png"
+/// \example archetypes/scalars_row_updates title="Update a scalar over time" image="https://static.rerun.io/transform3d_column_updates/2b7ccfd29349b2b107fcf7eb8a1291a92cf1cafc/1200w.png"
+/// \example archetypes/scalars_column_updates title="Update a scalar over time, in a single operation" image="https://static.rerun.io/transform3d_column_updates/2b7ccfd29349b2b107fcf7eb8a1291a92cf1cafc/1200w.png"
 table Scalar (
+  "attr.rerun.deprecated": "Use `Scalars` instead.",
```

Comment:
I think we should mandate that we include _when_ this was deprecated ("Deprecated since Rerun 0.23")